namerd/storage/k8s/src/main/scala/io/buoyant/namerd/storage/K8sDtabStore.scala

```diff
@@ -38,6 +41,8 @@ class K8sDtabStore(client: Http.Client, dst: String, namespace: String)
   val api: NsApi = Api(client.newService(dst)).withNamespace(namespace)
   val watchApi: NsApi = Api(client.newService(dst)).withNamespace(namespace)
 
+  private[this] var instrumentedDtabStorage: mutable.Seq[InstrumentedDtabStorage] = mutable.Seq[InstrumentedDtabStorage]()
```

Comment:
I don't think this needs to be a var OR a mutable.Seq.  What if this was just a val and `val act` was defined in terms of this?

```
val instrumentedDtabStorage = {
  val watchState = ...
  val act = watchApi.dtabs.activity...
}
val act = instrumentedDtabStorage.underlying
```

Reply:
That's a good point. I can make that change.